Dynatrace/dynatrace-operator
# Deprecate feature flags as they are switched to enabled by default

## Description

With this the following feature flags will be marked as deprecated in the validation webhook:

- ActiveGate Auth Token
- ActiveGate ReadOnly FS
- ActiveGate Raw Image
- ActiveGate Updates
- ActiveGate Automatic K8S Monitoring
- Hosts Request
- OneAgent ReadOnly FS
- Webhook Reinvocation Policy
- Metadata Enrichment
- Automatic Injection
- Label Version Detection
- Injection Failure Policy

Some of those features were already enabled by default, some are now switched to enabled by default.

## How can this be tested?

There are unit tests added, but otherwise deploy a dynakube with no feature flags set and check whether those features that are listed above are enabled.

## Checklist
